src/main/java/org/owasp/esapi/Encryptor.java

```diff
@@ -162,8 +162,8 @@ CipherText encrypt(SecretKey key, PlainText plaintext)
      * </p>
      * @param ciphertext The {@code CipherText} object to be decrypted.
      * @return The {@code PlainText} object resulting from decrypting the specified
-     *            ciphertext. Note that it it is desired to convert the returned
-     *            plaintext byte array to a Java String is should be done using
+     *            ciphertext. Note that the returned plaintext byte array
```

Comment:
This works, but I probably would have just fixed the actual typo here and changed the first 'it' to 'if' as was originally intended.

Reply:
Resolving this since your additional commit just revised this to 'if'.